pkg/events/events.go

```diff
@@ -56,6 +56,8 @@ const (
 	EventReasonSyncWorkloadSucceed = "SyncSucceed"
 	// EventReasonWorkDispatching indicates that work is dispatching or not.
 	EventReasonWorkDispatching = "WorkDispatching"
+	// EventReasonBindingScheduling indicates that binding is scheduling or not.
+	EventReasonBindingScheduling = "BindingScheduling"
```

Comment:
Can you remind me the benefit of having this event?

Reply:
The user can have an intuitive message to know what is going on about the scheduling process.

Reply:
OK.

Seems we should put it on the `const block` below where defines the events for ResourceBinding.

Reply:
It makes sense: )